packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/method_channel_google_maps_flutter.dart

```diff
@@ -42,9 +42,9 @@ class MethodChannelGoogleMapsFlutter extends GoogleMapsFlutterPlatform {
   /// platform side.
   ///
   /// The returned [Future] completes after listeners have been notified.
-  Future<void> updateMapOptions(Map<String, dynamic> optionsUpdate) async {
+  Future<void> updateMapOptions(Map<String, dynamic> optionsUpdate) {
     assert(optionsUpdate != null);
-    await _channel.invokeMethod<void>(
+    _channel.invokeMethod<void>(
```

Comment:
Yeah, all the `Future<void>` need to be `async` and `await` for the `_channel` to be done for all of this to work. Aren't you seeing some bugs because of this?

(I'll push some code fixing these, my bad!)

Reply:
I've fixed this by `return`ing the call to the _channel.